INSTALL.md

```diff
@@ -70,7 +70,7 @@ $ (cd jefferson && sudo python setup.py install)
 # Install ubi_reader to extract UBIFS file systems
 $ sudo apt-get install liblzo2-dev python-lzo
 $ git clone https://github.com/jrspruitt/ubi_reader
-$ (cd ubi_reader && sudo python setup.py install)
+$ (cd ubi_reader && poetry install)
```

Comment:
If `ubireader` _requires_  `poetry install`, it is broken.

This should just be

```suggestion
$ pip install ./ubi_reader
```
to have Pip do the right PEP 621 thing.